src/Features/CSharp/Portable/BraceCompletion/CharLiteralBraceCompletionService.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.BraceCompletion;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.BraceCompletion
+{
+    [Export(LanguageNames.CSharp, typeof(IBraceCompletionService)), Shared]
+    internal class CharLiteralBraceCompletionService : AbstractBraceCompletionService
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public CharLiteralBraceCompletionService()
+        {
+        }
+
+        protected override char OpeningBrace => SingleQuote.OpenCharacter;
+
+        protected override char ClosingBrace => SingleQuote.CloseCharacter;
+
+        public override Task<bool> AllowOverTypeAsync(BraceCompletionContext braceCompletionContext, CancellationToken cancellationToken)
+            => SpecializedTasks.True;
```

Comment:
doc this.  i think i can understand why it is the cae, but it's not totally clear (esp. for a future reader).